### ISSUE TITLE
dashboard becoming unmanageable with 34+ games

### ISSUE THREAD

**Issue by Ragzouken** [itchio/itch.io#658]
personally for me it'd be useful to have the list sorted by my last activity (updating them etc)
maybe it'd also help to have them in two columns with the analytics summaries sitting on top instead of beside

**Comment by leafo** [comment]
I'm trying to think of modifications to the page that would be friendly to people with 1 game and 50 games. Sorting options is a good start. I'm worried about analytics on the top pushing things down.

Do you think something like this is an improvement?

![](http://leafo.net/shotslp/2017-06-07_14-34-16.png)

**Comment by Ragzouken** [comment]
imo yes, that's an improvement (nearly halving the height of the page)

**Comment by leafo** [comment]
The problem with that design is that we use the sidebar to put notifications on, and if it's turned into the graphs on the top then it's hard to reuse those components. An another approach could be making a very wide screen variation that turns the game column into two game columns

**Comment by fasterthanlime** [comment]
I believe changing the default sort (as @Ragzouken pointed out) and providing a quick filter by name would help a lot already (and be less trouble that a two-column layout)

**Comment by leafo** [comment]
is filter by name a useful thing when you can use your browser's built in serach?

**Comment by fasterthanlime** [comment]
> is filter by name a useful thing when you can use your browser's built in serach?

I believe it is - less than a good default sort, but still :) browser built-in search make for a bunch of false positives.

**Comment by Ragzouken** [comment]
yeah the sorting thing is probably enough tbh - the main friction i'm facing is that i have 32 old games i will never want to update and i need to scroll through to find my latest project each time. other useful alternatives might be the ability to "archive" old games (in the dashboard only), or to "star" / "favourite" some to appear at the top

and yeah personally i prefer filter to browser search whenever available

**Comment by Ragzouken** [comment]
also, it'd be great to move the "create new project" button to the top, atm i have to scroll the full length of the page to find it!

**Comment by YellowAfterlife** [comment]
I have 48 items total on my dashboard and made this userstyle a while ago:
https://userstyles.org/styles/138747/itch-io-dashboard-grid (CC0)
I think displaying (or having an option to) the projects in a small grid is a worthwhile idea, since it's often enough quicker to navigate by thumbnails than by title text.
